openpype/modules/python_console_interpreter/window/widgets.py

```diff
@@ -627,8 +628,45 @@ def add_tab(self, tab_name, index=None):
     def showEvent(self, event):
         self._line_check_timer.start()
         super(PythonInterpreterWidget, self).showEvent(event)
+        # First show setup
+        if self._first_show:
+            self._first_show = False
+            self._on_first_show()
+
         self._output_widget.scroll_to_bottom()
 
+    def _on_first_show(self):
+        # Change stylesheet
+        self.setStyleSheet(load_stylesheet())
+        # Check if splitter size raio is set
+        # - first store value to local variable and then unset it
+        splitter_size_ratio = self._splitter_size_ratio
+        self._splitter_size_ratio = None
```

Comment:
It does seem weird that we're storing this value to `self` just so it's used this one time on first show and only then. I understand the `init_from_registry()` takes all settings from registry but nonetheless this feels odd.

Reply:
> It does seem weird that we're storing this value to self just so it's used this one time on first show and only then.

I agree but at the same time loading the data from registry twice is weirder. Since splitter sizes must be set after show and window resize must happen before show, it's just decision.